### PR TITLE
V1 fix issues

### PR DESCRIPTION
**Pull request by ace0625**
- Fix refresh data source issue from userlist
- Fix refresh control image issue on Ios device

### CODE REVIEW

File: ios/Yoda.xcodeproj/xcshareddata/xcschemes/Yoda.xcscheme [hunk 1]
```diff
@@ -66,9 +66,9 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
-      selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
-      selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      buildConfiguration = "Release"
```
Comment (kayla-kwon) [inline ios/Yoda.xcodeproj/xcshareddata/xcschemes/Yoda.xcscheme:69]: Congrats!

**Comment by kayla-kwon** [comment]
:shipit:

**Comment by kayla-kwon** [comment]
Oh u guys don't have this lol
LGTM

**Comment by kayla-kwon** [comment]
and don't have me..

**Comment by lyw0149** [comment]
kk your email is changed..

**Comment by kayla-kwon** [comment]
:shipit: ❤️

**Pull request merged**